pages/osx/ed.md

```diff
@@ -0,0 +1,25 @@
+# ed
+
+> The original Unix text editor.
+> See also: `awk`, `sed`.
+> More information: <https://www.gnu.org/software/ed/manual/ed_manual.html>.
```

Comment:
Do not support the same arguments in macOS?

Reply:
No, according to [this](https://www.freebsd.org/cgi/man.cgi?ed(1)) man page.